WooCommerce/Classes/ViewRelated/Orders/Order Creation/Taxes/NewTaxRateSelectorView.swift

```diff
@@ -156,5 +181,10 @@ extension NewTaxRateSelectorView {
         static let taxRatesSectionTitle = NSLocalizedString("Select a tax rate", comment: "Title for the tax rate selector section")
         static let editTaxRatesInWpAdminButtonTitle = NSLocalizedString("Edit tax rates in admin",
```

Comment:
Not from this PR, but for consistency should we also rename this to `Store Admin`, or `wp-admin`?

Reply:
Gotcha, thanks for clarifying! We seem to use the 3 of them (admin, wp-admin, store admin) indiscriminately between features, I'll raise some examples with design to see if we can get a common approach 👍

Reply:
> I'll raise some examples with design to see if we can get a common approach 👍

That would be great! thanks for that